pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[build-system]
+requires = [
+    "setuptools>=41.2",
+    "setuptools_scm[toml]>=1.15",
```

Comment:
Working, I'll revisit setuptools_scm but in general I avoid it in favor of `__init__.py` defining `__version__` and setting in `setup.cfg`.  When you get archives of repos (e.g. `.tar.gz` downloads) without the git history packaging can become difficult.

Reply:
Also good, no need to overcomplicate this for such a simple repo.